docs/success-guide/command_line_options.md

```diff
@@ -42,13 +42,13 @@ Note that only the users returned by the directory query and filter are consider
 | Command line option       ...........| Use when           |
 | ------------- |:-------------| 
 |   None                        |  No action desired on nonexistent users |
-|   `--remove-entitlements-for-nonexistent-users`\* |    Adobe account to remain but licenses and group <br>memberships are removed.  |
-|   `--remove-nonexistent-users`  |    Adobe account to remain but licenses, group memberships, and membership in this org to be removed   |
-|   `--delete-nonexistent-users`  |    Adobe account to be deleted: remove from PLCs and user groups and <br>from the org; account deleted and all storage and settings freed. |
-|   `--generate-remove-list f`    |  No action to be taken on the account.  User name written to file for later action. |
-|   `--generate-delete-list f`    |  No action to be taken on the account.  User name written to file for later action. |
+|   `--adobe-only-user-action remove-adobe-groups` |    Adobe account to remain but licenses and group <br>memberships are removed.  |
+|   `--adobe-only-user-action remove`  |    Adobe account to remain but licenses, group memberships, and membership in this org to be removed   |
```

Comment:
I thought we were trying to stay away from "org" as a concept?  Maybe something like "all licenses, group memberships, and visibility in the Admin Console are removed, but not stored assets."